test/clj/swarmpit/registry/client_test.clj

```diff
@@ -7,5 +7,5 @@
 
   (testing "dns error"
     (is (thrown-with-msg?
-          ExceptionInfo #"Registry failure: .* Name or service"
+          ExceptionInfo #"Registry failure: .*"
```

Comment:
The exception being thrown here is of java.net.UnknownHostException so the message is also different